platform/barefoot/sonic-platform-modules-bfn/debian/rules

```diff
@@ -3,20 +3,32 @@
 PACKAGE_NAME := sonic-platform-modules-bfn
 SCRIPT_SRC := $(shell pwd)/scripts
 CONFIGS_SRC := $(shell pwd)/configs
+BUILD_DIR := $(shell pwd)/build
+WHEEL_BUILD_DIR := $(BUILD_DIR)/wheel
 
 %:
 	dh $@
 
+override_dh_auto_build:
+	set -e
+	python2.7 setup.py bdist_wheel -d $(WHEEL_BUILD_DIR)
```

Comment:
We are currently in the process of transitioning all Python code from Python 2 to Python 3. We are now asking all vendors to build/install/copy both Python 2 and Python 3 versions of their sonic_platform package in preparation for the transition. Can you please update the PR to do this?

Reply:
Done